numpy/lib/npyio.py

```diff
@@ -835,6 +835,11 @@ def loadtxt(fname, dtype=float, comments='#', delimiter=None,
         the system default is used. The default value is 'bytes'.
 
         .. versionadded:: 1.14.0
+    maxlines : int, optional
+        Only read `maxlines` lines of the content. The default is to read all
+        lines.
+
+        .. versionadded:: 1.16.0
```

Comment:
Needs an explanation of interaction of `skiprows` and `maxlines`, "Read `maxlines` lines of content after `skiplines`. The default (`None`) is to ..."